roxie/ccd/ccdqueue.cpp

```diff
@@ -746,6 +778,11 @@ class RoxieQueue : public CInterface
             queueLength++;
             if (queueLength>maxQueueLength)
                 maxQueueLength = queueLength;
+            if (idle==0 && started < numWorkers)
```

Comment:
There is a minor race condition here.  If there is 1 idle thread, and multiple enqueue()s occur at the same time, no new threads will be started.